MnQ(doc) - update sqs node code sample (#1679)
scaleway/docs-content: serverless/messaging/api-cli/python-node-sqs.mdx

### Connect to SQS

> This tutorial use the `@aws-sdk/client-sqs` module which is the latests SDK available.

Reviewer: ```suggestion
Here, we use the `@aws-sdk/client-sqs` module, which is the latest SDK available.
```

> Import the required module

Reviewer: ```suggestion
Import the required module:
```
